irohad/validation/chain_validator.hpp

```diff
@@ -32,17 +32,17 @@ namespace iroha {
       // TODO andrei 16.10.18 IR-1761 Rename methods in validators
```

Comment:
This ToDo should be removed, but as the task is about SFV as well, it is not to be closed yet

Reply:
done